### DISCUSSION TITLE
cache logging (hit/miss/time) isn't shown anymore on 13.4.19 by default

### DISCUSSION THREAD

**Discussion by Lodimup** [vercel/next.js#54620] [App Router]
13.4.1 used to show state of cache either hit or miss and time it takes.

Is there a way to reenable this behaviour  on 13.4.19?

**Comment by asyarb** [comment]
From what I can tell, I think this is now enabled behind an experimental flag in `next.config.js` (as of 13.4.19):

```js
const nextConfig = {
  experimental: {
    logging: "verbose",
  }
}
```
